vidarr-server/src/main/resources/db/migration/V0013__niassa_accession_index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX niassa_index ON analysis ((labels ->> 'niassa-file-accession'));
```

Comment:
Nice! Though since we also have workflow run SWIDs in the workflow_run table, perhaps this index could be renamed to `niassa_file_swid_index`?

Reply:
Done!